iosHyperskillApp/iosHyperskillApp/Sources/Modules/TrackSelection/Details/Views/TrackSelectionDetailsView.swift

```diff
@@ -107,23 +107,8 @@ private extension TrackSelectionDetailsView {
             DispatchQueue.main.async {
                 stackRouter.popToRootViewController()
             }
-        case .home(let navigateToHomeViewAction):
-            switch navigateToHomeViewAction.command {
-            case .backto:
-                TabBarRouter(
-                    tab: .home,
-                    popToRoot: true
-                )
-                .route()
-
-                DispatchQueue.main.async {
-                    stackRouter.popToRootViewController()
-                }
-            case .newrootscreen:
-                viewModel.doNavigateToHomeAsNewRootScreenPresentation()
```

Comment:
Please delete all iOS-related code, start from `viewModel.doNavigateToHomeAsNewRootScreenPresentation()` and search all next symbols